products/pluto/dbt_project.yml

```diff
@@ -11,3 +11,4 @@ tests:
 
 vars:
   version: "{{ env_var('VERSION') }}"
+  version_type: "{{ env_var('VERSION_TYPE') }}"
```

Comment:
This `dbt` var `version_type` isn't actually used in code.

I define it here nonetheless because this env var is used in a dbt CLI command (see last line of `products/pluto/pluto_build/07_custom_qaqc.sh` file), and `dbt` checks that this env variable is present before running any code - very convenient.